new-relic-deployment-marker/pipe.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+## Required Parameters
+NEW_RELIC_API_KEY=${NEW_RELIC_API_KEY}
+NEW_RELIC_APPLICATION_ID=${NEW_RELIC_APPLICATION_ID}
+DEPLOYMENT_REVISION=${DEPLOYMENT_REVISION}
+
+## Optional Parameters
+DEPLOYMENT_USER=${DEPLOYMENT_USER:='bitbucket.pipeline'}
+NEW_RELIC_REGION=${NEW_RELIC_REGION:='US'}
```

Comment:
Should the region be US or should this be variable for Europe, Asia, etc?

Reply:
Default is US in Newrelic, but it depends where the data is hosted (can either be EU or US) - Only added this so I could test it locally as I setup an account in EU